Vagrantfile

```diff
@@ -17,6 +17,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box_version = "> 0.3.2.1714"
 
   config.vm.provision :shell, :inline => $bootstrap, :privileged => true
+  # Use the git configuration from the host in the VM
+  config.vm.provision "file", source: "~/.gitconfig", destination: ".gitconfig"
```

Comment:
I'm not sure why this is irrelevant or detrimental to using this as a development box. In fact it would have saved me time in the past.